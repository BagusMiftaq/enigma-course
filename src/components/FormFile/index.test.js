import {render} from "@testing-library/react";

import FormFile from "./index";

test("Should render FormFile correctly", ()=>{
    const mockProps = {label: "this is label", placeholder: "this is placeholder",
                        value: "this is value", onChange: ()=>{}, disabled: false,
                        id : "id"}

    const {asFragment} = render(<FormFile {...mockProps}/>);

    expect(asFragment()).toMatchSnapshot();
})