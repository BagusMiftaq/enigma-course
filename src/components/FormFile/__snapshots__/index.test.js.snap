// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render FormFile correctly 1`] = `
<DocumentFragment>
  <div
    class="mb-3"
  >
    <label
      class="form-label"
    >
      this is label
    </label>
    <input
      class="form-control is-valid"
      placeholder="this is placeholder"
      type="file"
    />
    <small
      class="form-text"
      id="id"
    >
      this is value
    </small>
  </div>
</DocumentFragment>
`;
